examples/tutorial_derive/03_02_option_mult.md

```diff
@@ -15,6 +15,9 @@ name: []
 $ 03_02_option_mult_derive --name bob
 name: ["bob"]
 
+$ 03_02_option_mult_derive --name bob --name john
```

Comment:
Could up update `examples/tutorial_builder/03_02_option_mult.md`?  These files are intended to be kept in sync

Reply:
I haven't learned the [Builder tutorial](https://docs.rs/clap/latest/clap/_tutorial/chapter_0/index.html) yet.
So, I'll update `examples/tutorial_builder/03_02_option_mult.md` after I finish learning it.

Reply:
Don't really need to learn it as this is just running the commands and the commands should have the same behavior.

Reply:
Ok, Understood. Thanks for the clarification.

Reply:
I fixed `examples/tutorial_builder/03_02_option_mult.rs` as it seems to be wrong.